semantic_conventions/trace/rpc.yaml

```
@@ -33,3 +33,14 @@ groups:
           - net.peer.ip
           - net.peer.name
         - include: network
+    - id: rpc.grpc
+      prefix: rpc.grpc
+      extends: rpc
+      brief: 'Call-level attributes for gRPC'
+      attributes:
+        - id: status_code
+          type: number
```

Hmm, this would logically be an integer enum but I don't think we support that yet, @thisthat ?

---

I didn't see any... https://github.com/open-telemetry/build-tools/issues/19

---

Sorry for the late response :sweat_smile: 
It is already possible to declare an integer enum with the current tool :sunglasses: 
```suggestion
          type:
            allow_custom_values: false
            members:
              - id: OK
                value: 0
              - id: CANCELLED
                value: 1
              ....
```

---

let's add this?